source/routing/raptor.cpp

```diff
@@ -224,15 +248,81 @@ make_starting_points_snd_phase(const RAPTOR& raptor,
                 count,
                 (clockwise ? working_labels.dt_pt(a.first) + walking_t
                            : working_labels.dt_pt(a.first) - walking_t),
-                walking_t
+                walking_t,
+                false
             };
-            res.add(starting_point);
+            overfilter.add({res.size(), starting_point});
+            res.push_back(starting_point);
         }
     }
 
+    for (const auto& pair : overfilter) {
+        res[pair.first].has_priority = true;
+    }
+
+    std::sort(res.begin(), res.end(), CompSndPhase(clockwise)); // most interesting solutions first
+
     return res;
 }
+
+Journey convert_to_bound(const StartingPointSndPhase& sp,
+                         uint32_t lower_bound_fb,
+                         uint32_t lower_bound_conn,
+                         bool clockwise) {
+    Journey journey;
+    journey.sections.resize(sp.count);
+    journey.sn_dur = navitia::time_duration(0, 0, sp.fallback_dur + lower_bound_fb, 0);
+    uint32_t nb_conn = (sp.count >= 1 ? sp.count - 1 : 0);
+    if (clockwise) {
+        journey.arrival_dt = sp.end_dt;
+        journey.departure_dt = sp.end_dt - journey.sn_dur.seconds() - nb_conn * lower_bound_conn;
+    } else {
+        journey.arrival_dt = sp.end_dt + journey.sn_dur.seconds() + nb_conn * lower_bound_conn;
+        journey.departure_dt = sp.end_dt;
+    }
+
+    journey.transfer_dur = navitia::seconds(120 * sp.count + nb_conn * lower_bound_conn);
```

Comment:
The possible use is : `120_s * sp.count +  navitia::seconds(nb_conn * lower_bound_conn);
Not so nice